src/silverlabnwb/silverlab.ophys.yaml

```
@@ -14,19 +14,18 @@ groups:
     dtype: int
     shape:
     - 2
-    doc: ' the 2d imaging frame size in voxels'
+    doc: the 2d imaging frame size in voxels
   - name: imaging_mode
     dtype: text
     doc: the acquisition mode for the experiment; pointing = single-voxel ROIs, miniscan
       = 2d rectangular ROIs, volume = 3d cuboid ROIs
   datasets:
   - neurodata_type_def: ZplanePockelsDataset
-    name: pockels
     shape:
     - null
     - 4
-    doc: "pockels data set, recording calibration data for focusing at different z-planesin\
-      \ four columns: Z offset from focal plane (micrometres), normalised Z,'Pockels'\
+    doc: "pockels data set, recording calibration data for focusing at different z-planes\
+      \ in four columns: Z offset from focal plane (micrometres), normalised Z, 'Pockels'\
       \ i.e. laser power in %, and z offset for drive motors"
```

Is it possible to use a multiline string like we did in [the old `metadata.yaml` before adding comments](https://github.com/SilverLabUCL/PySilverLabNWB/blob/366a24211ec4b1e1dd6dbe7a9d5692da9a2f751f/src/silverlabnwb/metadata.yaml)? Such as:
```suggestion
    doc: >
      pockels data set, recording calibration data for focusing at different z-planes
      in four columns: Z offset from focal plane (micrometres), normalised Z, 'Pockels'
      i.e. laser power in %, and z offset for drive motors
```
If this works, it would also remove the escaping as mentioned in #50.

---

🤔 this file is intended to be created by `generate_extended_schema` (and should have been up-to-date with it at all times, although it clearly wasn't (a typo fix didn't reach this)). I don't know whether we could get the schema builder from pynwb to generate multiline comments?

---

Right, of course! Disregard that comment then.